language/de/cli.php

```diff
@@ -160,6 +162,10 @@
 
 	'CLI_USER_ADD_SUCCESS'		=> 'Benutzer %s erfolgreich angelegt.',
 	'CLI_USER_DELETE_CONFIRM'	=> 'Soll „%s“ wirklich gelöscht werden? [y/N]',
+	'CLI_USER_DELETE_ID_CONFIRM'	=> 'Bist du sicher, dass du die Benutzer mit den IDs „%s“ löschen möchtest? [y/N]',
+	'CLI_USER_DELETE_ID_SUCCESS'	=> 'Die Benutzer-IDs wurden erfolgreich gelöscht.',
```

Comment:
Auch wenn es der englischen Übersetzung entspricht: Besser ist "Die Benutzer mit den angegebenen IDs wurden gelöscht." - es werden ja die ganzen Benutzer und nicht nur deren IDs gelöscht ;-)

Reply:
Was hältst du dann von "Die angegebenen Benutzer wurden gelöscht."? Dass der Admin vorher IDs eingegeben hat weiß er vermutlich eh selber...

Reply:
Die angegebenen Benutzer wurden gelöscht klingt besser.

Mein Nachtrag kam wohl zu spät: bei allen Änderungen ist ein d zu viel in Addresse - sollte Adresse heißen.

Reply:
Umgesetzt.